modules/linear_boltzmann_solvers/a_lbs_solver/acceleration/diffusion_mip_solver.cc

```diff
@@ -438,7 +433,7 @@ DiffusionMIPSolver::Assemble_b_wQpoints(const std::vector<double>& q_vector)
       {
         const int64_t imap = sdm_.MapDOF(cell, i, uk_man_, 0, g);
         double entry_rhs_i = 0.0; // entry may accumulate over j
-        if (source_function.empty())
+        if (!source_function_)
```

Comment:
Convention... Most everywhere we have tried to use `not` `and` `or`. We should stick to that if possible.